### PR TITLE
openshift event watcher/reporter

### PR DESCRIPTION
**Pull request by joelddiaz**
Use /usr/bin/oc to watch events stream in as json.
Wake up a thread every minute to count and report on the events (via --watch-for parameter).

### CODE REVIEW

**Comment by twiest** [comment]
:+1:

File: scripts/monitoring/cron-event-watcher.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import argparse
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+import json
+from Queue import Queue
+import re
+import subprocess
+import threading
+import time
+
+EVENT_PERIOD = 60 # seconds
```
Comment (kwoodson) [inline scripts/monitoring/cron-event-watcher.py]: Any reason not to put this into the config file?  Seems like we could potentially want to change the interval without having to rebuild.

**Comment by joelddiaz** [comment]
@kwoodson: updated PR to allow for passing in custom event timeout period

**Comment by kwoodson** [comment]
:+1:

**Comment by twiest** [comment]
:+1:

**Pull request merged**